src/rdo.rs

```diff
@@ -524,8 +524,8 @@ pub fn rdo_cdef_decision(sbo: &SuperBlockOffset, fi: &FrameInvariants,
         let mut err:u64 = 0;
         for by in 0..8 {
             for bx in 0..8 {
-                let bo = sbo.block_offset(bx, by);
-                if bo.x+bx < bc.cols && bo.y+by < bc.rows {
+                let bo = sbo.block_offset(bx<<1, by<<1);
```

Comment:
looks like `bx` and `by` are always used `<<1` you could name something like `scaled_bx` and `scaled_by` and use them everywhere.

Reply:
See https://github.com/xiph/rav1e/pull/444/files#diff-74a7e6f07dbcb44c01aceadb6825046dR533
and https://github.com/xiph/rav1e/pull/444/files#diff-74a7e6f07dbcb44c01aceadb6825046dR537